xonsh/completer.py

```diff
@@ -78,6 +78,21 @@ def _normpath(p):
 
     return p
 
+def alias_expand(line):
```

Comment:
This function seems like it would be better suited to being a method on the `built_ins.Aliases` class.  (Also the Aliases class should probably be moved to aliases.py, but that is another story.)